apps/analysis/models.py

```diff
@@ -50,6 +50,16 @@ class AnalysisPillar(UserResource):
     def __str__(self):
         return self.title
 
+    def clone_pillar(self, title):
+        pillar_cloned = copy.deepcopy(self)
+        pillar_cloned.pk = None
+        pillar_cloned.client_id = None
+        pillar_cloned.title = f'{title} (cloned)'
```

Comment:
Do we need to append `(cloned)` to the user-provided title? @AdityaKhatri

Reply:
Can we have the logic where if its the same name we append the cloned status?

Reply:
Let's not append `(cloned)`. Since we need to create/store the cloned_from value, the client can show if it was cloned or not.

Reply:
Updated in `fa0ef22`